content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp

```diff
@@ -0,0 +1,51 @@
+<%--
```

Comment:
Why is this a JSP? Is this somethin we overlay from a parent component?

Reply:
I added a new node here https://github.com/adobe/commerce-cif-connector/blob/4d04e8f1564e68564b22515f327c14639090570b/content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/_cq_dialog/.content.xml#L48

Reply:
But is that node a new component. If so, why are we using JSPs and not HTML?

Reply:
I use some Java code to get the path of referenced configuration [here](https://github.com/adobe/commerce-cif-connector/blob/19a2bb3e957fd6dce7f65b0b0a89cec4819e90c4/content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp#L23-L38)
I'm not sure I can do this with HTL

Reply:
You can't, you have to use a Sling Model, this is hot it works.

Reply:
@dplaton switched to HTL